lightning/src/util/config.rs

```diff
@@ -120,11 +120,10 @@ pub struct ChannelHandshakeLimits {
 	///
 	/// Default value: 546, the current dust limit on the Bitcoin network.
 	pub min_dust_limit_satoshis: u64,
-	/// Maximum allowed threshold above which outputs will not be generated in their commitment
-	/// transactions.
-	/// HTLCs below this amount plus HTLC transaction fees are not enforceable on-chain.
+	/// Maximum allowed dust_limit_satoshis for their commitment+HTLC transactions, above which
```

Comment:
Why did you change this? The new text seems even less readable - making reference to the particular name of the variable in the spec doesn't describe what it means, and is rather bad documentation.

Reply:
Because comment is wrong, as far as I understand.  Outputs will be generated if they are above the `dust_limit_satoshis`, but we won't accept channel if proposed value is above this "maximum allowed threshold". 

On `dust_limit_satoshis`, its meaning is described in variable above `min_dust_limit_satoshis`, to which name proximity is enough to rely on a reader going through its comment too ?

Reply:
Updated to make it clearer tho